### PR TITLE
backport from forks

### PR DESCRIPTION
**Pull request by vankasteelj**
pick useful updates from forks

### CODE REVIEW

File: src/app/lib/device/chromecast.js [hunk 1]
```diff
@@ -88,36 +90,17 @@
...
         },
 
         forward: function () {
-            this.seek(30);
+            this._seek(this.get('loadedMedia').currentTime + 30);
         },
 
         backward: function () {
-            this.seek(-30);
+            this._seek(this.get('loadedMedia').currentTime - 30);
```
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: implement that in _seek ? or seek ?

File: src/app/lib/views/movie_detail.js [hunk 2]
```diff
@@ -7,7 +7,8 @@
 
         ui: {
             bookmarkIcon: '.favourites-toggle',
-            watchedIcon: '.watched-toggle'
+            watchedIcon: '.watched-toggle',
+            selected_lang: '.selected-lang'
```
Comment (xaiki) [inline src/app/lib/views/movie_detail.js]: no no no no

Comment (xaiki) [inline src/app/lib/views/movie_detail.js]: this belongs into the dropdown code.

File: src/app/lib/device/chromecast.js (hunk 1 above)
Comment (vankasteelj) [inline src/app/lib/device/chromecast.js]: no, read the 3 functions using _seek, it's not feasible

File: src/app/lib/views/movie_detail.js (hunk 2 above)
Comment (vankasteelj) [inline src/app/lib/views/movie_detail.js]: I'm backporting from forks, it does make things better than without it (hence why they added it)

File: src/app/lib/device/chromecast.js (hunk 1 above)
Comment (xaiki) [inline src/app/lib/device/chromecast.js]: then do seek, if you don't want to implement this i'll do it but don't complain i touch your branches...

**Pull request merged**